app/packages/core/src/plugins/SchemaIO/components/FileExplorerView/FileExplorerView.tsx

```diff
@@ -17,38 +15,6 @@ export default function FileExplorerView(props) {
 
   const fsInfo = useAvailableFileSystems();
 
-  if (!fsInfo.ready) {
-    return (
-      <FieldWrapper {...props}>
-        <CircularProgress size={16} />
-      </FieldWrapper>
-    );
-  }
-
-  if (fsInfo.error) {
-    return (
-      <FieldWrapper {...props}>
-        <ErrorView
-          schema={{ view: { detailed: true } }}
-          data={[
-            {
-              reason: "Failed to get available file systems",
-              details: fsInfo.error,
-            },
-          ]}
-        />
-      </FieldWrapper>
-    );
-  }
-
-  if (fsInfo.available === false) {
```

Comment:
Why is this removed?

Reply:
All of this is now moved into adornment of custom path textfield